### PR TITLE
Jaana CSSpt2

### PR DESCRIPTION
**Pull request by jaanatabalon**
Added button styling for activity, category, and onboarding pages
Added link styling on onboarding pages

Screen Shot 2022-03-08 at 11 06 14 PM
Screen Shot 2022-03-08 at 11 06 04 PM
Screen Shot 2022-03-08 at 11 05 56 PM

### CODE REVIEW

File: src/utils/styledLink.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    font-size: 13;
+    text-transform: uppercase;
+    text-decoration: underline;
+    letter-spacing: .05rem;
+    border: none;
+    background: transparent;
+    color: MediumBlue;
+`;
+
+export { StyledLink };
```
Comment (liu-caro) [inline src/utils/styledLink.js:15]: In the future, we should fix this to just be CSS if possible. Importing styled components package just for changing style of link is unnecessary.

Comment (jaanatabalon) [reply src/utils/styledLink.js:15]: Ok - got it. I was thinking though that more components could be styled using the styled-components package, so we could get more usage out of it. We can talk about that change together?

File: src/index.css [hunk 2]
```diff
@@ -37,3 +37,62 @@ code {
...
+
+.landingOnboardingButton {
+  background-color: var(--giraffeYellow);
+  font-family: 'Sofia Pro Soft Regular';
+  letter-spacing: 15%;
+  font-size: 13;
+  border: none;
+  padding: 10px 20px 10px 20px;
+  border-radius: 40px;
+  box-shadow: 0px 6px 6px #999999;
```
Comment (liu-caro) [inline src/index.css:49]: feel like a black drop-shadow would be nicer/less flashy for on top of landing image

**Comment by liu-caro** [comment]
I know not from this commit, but overall text sizes for body text, buttons, and links feels a bit small on all screen sizes.

File: src/index.css [hunk 3]
```diff
@@ -37,3 +37,62 @@ code {
...
+
+.relationshipSkillsCategory {
+  background-color: var(--tigerRed);
+}
+
+.selfManagementCategory {
+  background-color: var(--hippoPurple);
+}
+
+.categoryPageButton {
```
Comment (liu-caro) [inline src/index.css:73]: We should add a min-width to button so they stay the same width and make the text left-justified.

**Comment by liu-caro** [comment]
styled buttons have a animation on click that's kind of distracting, could be nice improvement to remove animation.

**Pull request merged**